doc/en/components/data-grid-row-paging.md

```diff
@@ -100,4 +130,12 @@ private onPageChanged = (pageNumber: number, data: any[]) => {
 };
 ```
 
+## React Pagination and Grid Interactions 
```

Comment:
use $Platform$ variable instead of "React"

Reply:
done